packages/animated/package.json

```diff
@@ -21,55 +22,41 @@
     "module.js"
   ],
   "scripts": {
-    "clean": "rimraf ./dist",
     "prebuild": "npm run clean",
     "build": "npm run build:dev && npm run build:prod",
     "build:dev": "cross-env NODE_ENV=development rollup -c",
     "build:prod": "cross-env NODE_ENV=production rollup -c",
+    "clean": "rimraf ./dist",
     "test": "jest --silent",
     "test:watch": "jest --watch"
   },
-  "sideEffects": false,
+  "dependencies": {
+    "@pixi/react": "*"
+  },
```

Comment:
I believe this is one case that should be a peer dependency shouldn't it? It adds additional functionality on top of `pixi-react`

Reply:
Oh I thought this package was meant to be used instead of `pixi-react`

So you would only install `@pixi/react-animated` not both?
Maybe I misunderstood

Reply:
No the intention is you install both of them together. `@pixi/react-animated` is a very small wrapper around the normal lib.